docs/methodology/llm_inference.md

```diff
@@ -344,7 +417,12 @@ We aim at covering the largest scope possible when assessing the environmental i
 Please cite **GenAI Impact** non-profit organization and **link to this documentation page**. 
 
 ```bibtex
-Coming soon...
+"""
+@software{ecologits,
+  author = {Samuel Rincé, Adrien Banse, Vinh Nguyen, Luc Berton, and Chieh Hsu},
+  publisher = {GenAI Impact},
+  title = {EcoLogits: track the energy consumption and environmental footprint of using generative AI models through APIs.},
+}"""
```

Comment:
The citation block was for the paper about the methodology, but we never wrote it (at least for now). I think we can simply point to the GitHub repository to get the citation. I think @adrienbanse set this up.